README.md

```diff
@@ -1,5 +1,64 @@
-# curriculum-cr-training-contact-form
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This repo host an initial code for the tasks performed during the Code Reviewers training.
+# contact-form-review
 
-Application is based on this [Figma design](https://www.figma.com/file/t3EJUCAEViw3QasuJLPLVT/Microverse-Student-Potfolio-Templates-Main?node-id=1%3A1471).
+> contact-form-review: contact form of portfolio.
+
+![screenshot](./media/png/app_screenshot.png)
```

Comment:
- [ ] Kindly replace this as this screenshot is for the desktop version contact form and the project requirements are for the mobile version contact form.